src/components/Dashboard/Dashboard.jsx

```diff
@@ -4,10 +4,18 @@ import NavBar from './Header/Header';
 import MainContainer from './MainContainer/MainContainer';
 
 export default class Dashboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoggedIn: props.isLoggedIn,
+    };
+  }
+
   render() {
+    const isLoggedIn = this.state
```

Comment:
This is not proper destructuring. Please use this pattern:
`const { isLoggedIn } = this.state;`